traffic_ops/traffic_ops_golang/cachegroup/cachegroups.go

```diff
@@ -37,11 +37,13 @@ import (
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/dbhelpers"
 
-	"github.com/go-ozzo/ozzo-validation"
+	validation "github.com/go-ozzo/ozzo-validation"
 	"github.com/jmoiron/sqlx"
 	"github.com/lib/pq"
 )
 
+var ErrDuplicateExist = errors.New("cachegroup name already exist, give different name")
```

Comment:
This should not be exported

Reply:
also, grammar nit:

<del>exist</del> -> <ins>exists<ins>

<del>give different name</del> -> e.g. <ins>please choose a different name</ins> or similar